### ISSUE TITLE
Change club transfer offer in Pro Career/Player Career gameplay

### ISSUE THREAD

**Issue by Elboor** [xAranaktu/FIFA-18---Career-Mode-Cheat-Table#18]
Hello,

Tried to find it myself but having no luck, perhaps my method isn't very good.

Basically when you play as 'Be a pro/created player' in career mode I'd like to be able to change the destination/bidding club that has offered me a transfer as a player in career mode.

My method was searching for the wage offer (example: £100,000) then trying to locate a possible teamid as used in the playerdata/player edit and could find something that resembles the teamid (i.e : Arsenal 1, Man city 10, Man utd, 11) but when changing this value It kept shipping me to the club that originally bid for my player.

No idea if this the approach needed don't really know what I'm doing.

Thanks

**Comment by xAranaktu** [comment]
You approach was correct, you have just found the wrong address.

That's what I did:
I've started new career as Mesut Özil (ID: 176635). I've requested a transfer, and after a few days I've received offer from Manchester United (ID:11) which offered me a wage of $325 000 per week.
I've scanned for 325000 in CE and it found 15 addresses. Last 3 are probably inalid, so I've needed to check 12 address one by one.
![](https://i.imgur.com/deSL1ZD.png)

11th address looks promising:
![](https://i.imgur.com/VMAJ7Yk.png)

I've changed "TeamID" value from screen above and it's looks like it works:
![](https://i.imgur.com/c6jCQM6.png)

After accepting offer:
![](https://i.imgur.com/REFgHW9.png)
Looks like it works fine.

So, I decided to check what access the "TeamID address from 2nd screen:
![](https://i.imgur.com/CMrFxUB.png)

Unfortunately, it looks like the game have 2 almost exactly the same functions, so I needed to go deeper to cheat this, but anyway, this script should work:
You just need to replace "dd (int)241" with any other team id. 241 is [FC Barcelona](https://fifatracker.net/teams/241).
[SCRIPT](https://pastebin.com/B318Z9wf)

Just in case you don't know, you can grab teamids from here:
[FIFA Tracker - Teams](https://fifatracker.net/teams/)

**Comment by JohnnySborra** [comment]
Hey, just to say that this works perfectly. For those who play career mode extensively (I have about 750 hours in the game, most of them are playing Careers, either Player or Manager) this is a God-send.

Basically make sure you scan the wage address used for the actual offer, not just the email messages. There will be two wage addresses, one used in the email for "show" only, another used in the actual offer which will allow you to edit the team offer.

I donated to Aranaktu because he does such great work for the community, the game developers only care about FUT, and that mode is completely shit. Edits like these make the game worth playing again.

**Comment by BintouNdy** [comment]
hi i have added the script but can't get it to work (impossible to toggle the script). I appreciated what you do for carreer players and hope you have the time to respond.

**Comment by xAranaktu** [comment]
Script is outdated, so it will not work. You just simply need to follow the tutorial and create your own script.

**Comment by Arcanine878** [comment]
I can't find the values in my career mode I search it up and there's like 200 values do you have an updated script maybe to make it easier?

**Comment by xAranaktu** [comment]
No, I don't. To get less results just start a new career as a high rated player (90+) -> find right address -> make script -> load your main save and edit offer there.

**Comment by Arcanine878** [comment]
I'm having trouble finding the correct address how would I find it? there seems to be many many things to look through

**Comment by xAranaktu** [comment]
Trial and error method. Search for addresses and try to edit them one by one.

**Comment by caseyogutu** [comment]
Hi guys, first a big thank you to xAranaktu, your trainer has made the game more interesting. Second, I was wondering if it was possible to force a team to bid for your player using the same method in player career mode?

**Comment by victorsarbu** [comment]
Hello! I try to edit an offer in carreer mode as manager but when I click on it nothing happens. Can you help me, please?

**Comment by NotturnoVolpe** [comment]
Hey, thanks a lot for the CT they are GREAT !! But i can't find the team id, and i dunno how to make or run a script on CE. It's driving me nuts cuz i can't transfer to a team a want, any help would be lovely, thanks a lot !!

**Comment by chopebzz** [comment]
fucking genius, it works perfectly. thank

**Comment by Djordje-Nikolic** [comment]
Thank you so much, worked like a charm. As Johnny said, there will be two different addresses with the occurrence of the TeamID, change both so you are sure.
